app/src/main/java/com/android/sample/model/profile/ProfilesRepositoryFirestore.kt

```diff
@@ -38,13 +38,16 @@ open class ProfilesRepositoryFirestore @Inject constructor(private val db: Fireb
           (document.get("interests") as? List<*>)?.filterIsInstance<String>() ?: return null
       val activities =
           (document.get("activities") as? List<*>)?.filterIsInstance<String>() ?: return null
+      val likedActivities =
+          (document.get("likedActivities") as? List<*>)?.filterIsInstance<String>() ?: return null
       User(
           id = id,
           name = name,
           surname = surname,
           interests = interests,
           activities = activities,
-          photo = photo)
+          photo = photo,
+          likedActivities = likedActivities)
```

Comment:
The update to the documentToUser function to retrieve the likedActivities field from the Firestore document is a necessary step to integrate the "Like" feature into the user model.